### ISSUE TITLE
Long text is displayed wrong in the Bit playground

### ISSUE THREAD

**Issue by Mikilll94** [teambit/bit#4080]
### Describe the bug
If you have a long in the component, the component is displayed wrong in the playground and in the "compositions" section.

The text is contained in a single line although is should be splitted into multiple lines.

### Steps to Reproduce
1. Download this repro project:
[bit-overflow-repro.zip](https://github.com/teambit/bit/files/6278854/bit-overflow-repro.zip)
1. Run:

```
yarn
yarn start
```

to check how the component is displayed in the browser. It should be something like this:

![Screenshot 2021-04-08 at 13 08 00](https://user-images.githubusercontent.com/17653648/114032398-90583a80-987c-11eb-8ba4-56d13a05d4ec.png)

1. Then, launch the Bit playground:

```
bbit link
bbit start
```

The component is displayed like this:
![Screenshot 2021-04-08 at 15 11 49](https://user-images.githubusercontent.com/17653648/114032662-ce555e80-987c-11eb-91be-3c1cc0ca394c.png)

### Expected Behavior
In the Bit playground, text in the component should collapse into multiple lines.

There should be no difference in display between the component in the browser and the component in the Bit playground

### Specifications
* Bit version: 0.0.347
* Node version: v14.15.5
* yarn version: 1.22.10
* Platform: macOS 11.2.3

### Additional context
You can also check this bug here: https://bit.dev/miwasnie-org/bit-overflow-repro/test/overflow-repro

**Comment by KutnerUri** [comment]
playground wrap: happens because the whole playground is wrapped in a `<pre>` tag. Could be solved by replacing the tag (at MDX), or by adding `white-space: initial` to the react playground or the `snippet` component.
![Screen Shot 2021-04-08 at 16 19 06](https://user-images.githubusercontent.com/5400361/114033929-63a92080-9886-11eb-964e-d4c186d65290.png)

for the card - could be solved by setting `max-width: 100%` to the Composition Card, as already suggested by @benjgil, to handle 'full page' compositions.
![Screen Shot 2021-04-08 at 16 21 32](https://user-images.githubusercontent.com/5400361/114034394-d31f1000-9886-11eb-9f05-44220af728b5.png)
Can also remove the `flex: none` property.

Will make a PR when I get to it

**Comment by KutnerUri** [comment]
also - elements with `position: absolute` or `position: relative` can leak out of the composition card.

we need to add this to 'scope' these elements to the card:
```
position: relative;
transform: translateZ(0); // makes care become containing block for all descendants, incl. fixed position
```

**Comment by Mikilll94** [comment]
@KutnerUri 
Yeah, exactly. I have also noticed that elements with position absolute, fixed and relative can leak out of a a card.

**Comment by KutnerUri** [comment]
Closed by #4506, now included in 0.0.443

- prevent leakage from composition card and live-playground
- max-width: 100%

@Mikilll94 - FYI
